### ISSUE TITLE
No new data on coverage.moz.tools

### ISSUE THREAD

**Issue by darktrojan** [mozilla/code-coverage#2062]
It looks like something has broken between the ccov jobs running on taskcluster and the data appearing on the front end. The last data displayed there is from nearly a week ago.

**Comment by marco-c** [comment]
The coverage hook has been failing because the task that generated its Docker image expired. I'll need to make a new release to retrigger the Docker image generation task and everything will get back to working.

**Comment by marco-c** [comment]
```
[taskcluster 2023-09-15 21:20:28.681Z] Downloading artifact "public/code-coverage-bot.tar.zst" from task ID: WSfkLu2QQmyP0LpiwNWerQ.
[taskcluster 2023-09-15 21:20:28.812Z] Downloading artifact "public/code-coverage-bot.tar.zst" from task ID: WSfkLu2QQmyP0LpiwNWerQ.
[taskcluster 2023-09-15 21:20:28.948Z] Downloading artifact "public/code-coverage-bot.tar.zst" from task ID: WSfkLu2QQmyP0LpiwNWerQ.
[taskcluster 2023-09-15 21:20:29.107Z] Downloading artifact "public/code-coverage-bot.tar.zst" from task ID: WSfkLu2QQmyP0LpiwNWerQ.
[taskcluster:error] Pulling docker image has failed.
[taskcluster:error] Error: Error loading docker image. Could not download artifact "public/code-coverage-bot.tar.zst from task "WSfkLu2QQmyP0LpiwNWerQ" after 1 attempt(s). Error: Not Found
[taskcluster 2023-09-15 21:20:29.164Z] Unsuccessful task run with exit code: -1 completed in 3.083 seconds
```

**Comment by marco-c** [comment]
This should be fixed with 1ddd2097cece348fd771ac0363e901a8bec98719. I'll confirm and then close.
